components/map/MapItem.js

```diff
@@ -1,31 +1,81 @@
 import React from 'react'
-// import { Row, Col, Form, Button } from 'react-bootstrap/'
 // import PropTypes from 'prop-types'
 
+const openNav = () => {

```

Comment:
I think it might be better if we swap the value of this to be state and then use that value in the styling. That way we don't have to interact with the dom as there could be weirdness with the div being there or not being there as react rerenders the page when changes happen.

Something along these lines:

```javascript
//Setting the state
const [width, setWidth] = useState('0')

//Updating the function
const openNave = () => {
  setWidth('0')
}

// Or even do away with the function and set it on the onClick of the button?
<button onClick={() => setWidth('20%')}>

//Then set the width value as the state like this: https://nextjs.org/blog/styling-next-with-styled-jsx#dynamic-styles

```